CRM/Financial/BAO/Payment.php

```diff
@@ -439,11 +439,8 @@ public static function filterUntestedTemplateVariables($params) {
     ];
     // These are assigned by the payment form - they still 'get through' from the
     // form for now without being in here but we should ideally load
-    // and assign. Note we should update the tpl to use {if $billingName}
-    // and ditch contributeMode - although it might need to be deprecated rather than removed.
+    // and assign.
     $todoParams = [
-      'billingName',
```

Comment:
@eileenmcnaughton if a site hasn't updated their customised template / stored template does this mean that on upgrade the template will start effectively failing?

Reply:
@seamuslee001 I removed from this PR in favour of https://github.com/civicrm/civicrm-core/pull/33215